service/resource/flanneldv1/desired.go

```diff
@@ -2,8 +2,362 @@ package flanneldv1
 
 import (
 	"context"
+	"strconv"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/api/extensions/v1beta1"
+	apismeta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	"github.com/giantswarm/flannel-operator/service/keyv2"
+)
+
+var (
+	containersPrivileged       = true
+	failureThreshold     int32 = 2
+	healthEndpoint             = "/healthz"
+	initialDelaySeconds  int32 = 10
+	periodSeconds        int32 = 10
+	portBase                   = 21000
+	probeHost                  = "127.0.0.1"
+	successThreshold     int32 = 1
+	timeoutSeconds       int32 = 5
```

Comment:
Same copy/paste here. I only wanted to keep this locally in the package since it seems to be not of interest for other packages. This is a bit vague and other stuff might be moved around in the future. Good enough for me for now though.